SparkplugB feature addition to agent (#40)
Open-IoT-Service-Platform/oisp-iot-agent: README.md

For examples of how to use the 'oisp-agent' please see the [Examples](https://github.com/Open-IoT-Service-Platform/oisp-iot-agent/tree/master/examples) provided.

### Enabling SparkPlugB Standard 

> 1. To send data on SparkplugB standard data format, you need to enable it on agent and add its compatible sdk(oisp-sdk-js) to it. 
> 
> For this you need to make certain changed on the config.json file as below in mqtt connector section (added in config.json.template):

Reviewer: ```suggestion
1. To enable SparkplugB standard data format, you need to update the agent config as below:
```

Author: I agree with this comment

``` bash
 "mqtt": {
              "host": "emqx",
              "port": 1883,
              "qos": 1,
              "retain": false,
              "secure": false,
              "strictSSL": false,
              "retries": 5,
              "sparkplugB": true, 
              "version": "spBv1.0"            
          }
```
 - "SparkplugB":true -> enables the sparkplugB feature of agent
 - "version": "spBv1.0", -> version of spB standard, default is spBv1.0
 
 2.  CID is used as alias, as sparkplugB standard suggest unique id as alias in data metric element.
   Data message looks like below:

``` bash
    var cid = "0c574252-31d5-4b76-bce6-53f2c56b544d";
    var DataMessage = {
            timestamp: 12345,
            metrics: [{
                name : "temp",
                alias : cid,
                timestamp : 12345,
                dataType : "float",
                value: 123
            }],
            seq: 1
         };
```

3. Sending NGSI-LD compatible data over Mqtt-SpB
- Register devices for custom Relationship and Property catalog
- Message must be sent over SpB-Mqtt, which means flag in config must be like "sparkplugB": true,

> 

Author: yes can be ommited with explicitly stating in heading that it needs SparkplugB enabled
